frontend/src/pages/AccountSend.vue

```diff
@@ -469,15 +815,22 @@ function useSendForm() {
     return humanizeMinSendAmount(minSend);
   };
 
-  function isValidTokenAmount(val: string | undefined) {
+  function isValidTokenAmount(val: string | undefined, tokenInput?: TokenInfoExtended | null | undefined) {
     if (val === undefined) return true; // don't show error on empty field
     if (!val || !(Number(val) > 0)) return tc('Send.enter-an-amount');
-    if (!token.value) return tc('Send.select-a-token');
 
-    const { address: tokenAddress, decimals } = token.value;
+    let tokenToUse: TokenInfoExtended | null | undefined;
+    // Use batchSend token if it exists
+    if (tokenInput) tokenToUse = tokenInput;
+    // Otherwise use the current Send token
+    else tokenToUse = token.value;
+    if (!tokenToUse) return tc('Send.select-a-token');
```

Comment:
this feels clunky, the below should be equivalent right?
```suggestion
    const tokenToUse: tokenInput || token.value;
    if (!tokenToUse) return tc('Send.select-a-token');
```

Reply:
Indeed https://github.com/ScopeLift/umbra-protocol/pull/491/commits/f34b12a298489db5489c62ed096389876b209bde